dynamite_nsm/services/zeek/config.py

```diff
@@ -212,8 +212,8 @@ def add_worker(self, name, interface, host, lb_procs=10, pin_cpus=(0, 1)):
             pin_cpus = [str(cpu_n) for cpu_n in pin_cpus]
             self.node_config[name] = {
                 'type': 'worker',
-                'interface': interface,
-                'lb_method': 'pf_ring',
+                'interface': 'af_packet::' + interface,
+                'lb_method': 'custom',
                 'lb_procs': lb_procs,
                 'pin_cpus': ','.join(pin_cpus),
                 'host': host
```

Comment:
We may be able to accept the reasonable defaults for these but think we should set them explicitly?  Seems like we might want to customize them down the road.    

```
af_packet_fanout_id=23
af_packet_fanout_mode=AF_Packet::FANOUT_HASH
af_packet_buffer_size=128*1024*1024
```

Reply:
agreed